src/shogun/regression/svr/SVRLight.cpp

```diff
@@ -537,8 +537,8 @@ void SVRLight::update_linear_component_mkl(
 	}
 	else // hope the kernel is fast ...
 	{
-		float64_t* w_backup = SG_MALLOC(float64_t, num_kernels);
-		float64_t* w1 = SG_MALLOC(float64_t, num_kernels);
+		SGVector<float64_t> w_backup(num_kernels);
```

Comment:
you should rebase your local branch over the latest develop branch as this fix has been merged already

Reply:
I think the fix isn't on develop yet cause the same line 540 has an SG_MALLOC

Reply:
you'll have to pull the upstream develop and then rebase your branch over that... whatever remains that still needs fixing. but some of this is definitely in develop

Reply:
or maybe that's svmlight...? dunno make sure that you have rebased over the latest develop :)